app/brave_main_delegate.cc

```diff
@@ -195,8 +195,7 @@ bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
 
   // Brave variations
   std::string kVariationsServerURL = BRAVE_VARIATIONS_SERVER_URL;
-  command_line.AppendSwitchASCII(variations::switches::kVariationsServerURL,
```

Comment:
I don't think this should be removed, right? You want to set the value if it exists?

And below should be a `CHECK` actually, so that it completely crashes the build

Reply:
Perhaps a better check would be this instead:

```
std::string kVariationsServerURL = BRAVE_VARIATIONS_SERVER_URL;
command_line.AppendSwitchASCII(variations::switches::kVariationsServerURL,
                                                         kVariationsServerURL.c_str());
size_t pos = url->host().find("clientservices.googleapis.com");
CHECK(pos == std::string::npos);
```

Reply:
@bsclifton yes, switch shouldn't be removed. Just thinking if it makes sense to check `url->host().find("clientservices.googleapis.com")`. Assuming you meant `url` should be set to `kVariationsServerURL` this would never be `"clientservices.googleapis.com"` since it'll be either set by npmrc or empty?

Reply:
ah; yeah you're right, it would be empty

In that case, your original check (just as a CHECK instead of DCHECK) should work:
`CHECK_NE(kVariationsServerURL, nullptr);`